tools/pythonpkg/duckdb_python.cpp

```diff
@@ -207,6 +207,9 @@ PYBIND11_MODULE(DUCKDB_PYTHON_LIB_NAME, m) {
 	m.attr("threadsafety") = 1;
 	m.attr("paramstyle") = "qmark";
 
+	m.attr("EXPLAIN_STANDARD") = 0;
```

Comment:
Why not use `py::enum_` here?

Reply:
That's also possible, but like I said, that introduces another level:
```py
print(duckdb.sql('select 42').explain(duckdb.ExplainType.ANALYZE))
```

Reply:
Oh I actually that's not true
```py
>>> import duckdb 
>>> duckdb.ANALYZE
<ExplainType.ANALYZE: 1>
```